channels/uk.m3u

```diff
@@ -9,17 +9,17 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/akaal_tv/hls1_smart_akaa
 https://alaraby.cdn.octivid.com/alaraby/smil:alaraby.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="Arabic" tvg-logo="https://i.imgur.com/ooiJwD5.png" group-title="",Al Hiwar (1080p)
 https://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="BBC Arabic ARB" tvg-name="BBC Arabic ARB" tvg-country="UK" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oFGPG9Y.png" group-title="News",BBC Arabic (720p)
+#EXTINF:-1 tvg-id="BBC Arabic ARB" tvg-name="BBC Arabic ARB" tvg-country="UK;AR" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oFGPG9Y.png" group-title="News",BBC Arabic (720p)
```

Comment:
"AR" code of Argentina. For Arab countries there is a code "MENA" (Middle East and North Africa). A list of other supported region codes can be found here: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md#supported-region-codes

Reply:
@freearhey MENA includes Iran and Turkey which BBC Arabic does not broadcast to, this is an example of a channel that would benefit from an Arab Speaking Countries region.